### DISCUSSION TITLE
Badges have an expiration date or a renewal date?

### DISCUSSION THREAD

**Discussion by Ritik-Yadav** [drknzz/GitHub-Achievements#276] [Q&A]
Hello everyone!

Do these badges have an expiration date or a renewal date?
<img width="121" alt="Screenshot 2024-01-20 132821" src="https://github.com/drknzz/GitHub-Achievements/assets/60294338/945af1cf-37ac-4346-985b-75abcffa86f5">

**Comment by ghost** [comment]
Hello @Ritik-Yadav, is everything ok?
They have no expiration or renewal date, but you can choose whether or not to display them on your profile.

And also with each new mission completed, they increase.

**Reply by the1Riddle** [reply]
i second this :smile: :rocket:

**Comment by the1Riddle** [comment]
>Hello everyone!
> Do these badges have an expiration date or a renewal date?

No they dont have

**Comment by Boutzi** [comment]
Hopefuly not!
